arches/install/arches-templates/project_name/templates/email/download_ready_email_notification.htm

```diff
@@ -0,0 +1,28 @@
+{% load static %}
+{% load template_tags %}
+{% load i18n %}
```

Comment:
None of the strings in this template are wrapped in `trans`/ or `blocktrans` tags

Reply:
@chrabyrd Changed as requested